app/frontend/src/assets/scss/style.scss

```diff
@@ -2,14 +2,23 @@
  * BCGov Style
  * This globally extends the base Vuetify style
  */
-@import '~vuetify/src/styles/styles.sass';
+ @import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900');
+ @import url('https://fonts.googleapis.com/css?family=Material+Icons');
+ @import "~material-design-icons-iconfont/dist/material-design-icons.css";
+ @import "~font-awesome/css/font-awesome.css";
```

Comment:
I don't think we need FA on the global scope, it's only used by the bootstrap controls isn't it?

Reply:
that would be nice. i think you're right. i'll try moving this import to the component.